apps/studio/pages/project/[ref]/auth/advanced.tsx

```diff
@@ -0,0 +1,46 @@
+import { PermissionAction } from '@supabase/shared-types/out/constants'
+
+import { AdvancedAuthSettingsForm, SmtpForm, ThirdPartyAuthForm } from 'components/interfaces/Auth'
+import AuthLayout from 'components/layouts/AuthLayout/AuthLayout'
+import {
+  ScaffoldContainer,
+  ScaffoldDescription,
+  ScaffoldHeader,
+  ScaffoldTitle,
+} from 'components/layouts/Scaffold'
+import NoPermission from 'components/ui/NoPermission'
+import { GenericSkeletonLoader } from 'components/ui/ShimmeringLoader'
+import { useCheckPermissions, usePermissionsLoaded } from 'hooks/misc/useCheckPermissions'
+import type { NextPageWithLayout } from 'types'
+
+const AdvancedAuthSettings: NextPageWithLayout = () => {
+  const isPermissionsLoaded = usePermissionsLoaded()
+  // TODO: check if these permissions cover third party auth as well
+  const canReadAuthSettings = useCheckPermissions(PermissionAction.READ, 'custom_config_gotrue')
+
+  return (
+    <>
+      <ScaffoldContainer>
+        <ScaffoldHeader>
+          <ScaffoldTitle>Advanced</ScaffoldTitle>
+        </ScaffoldHeader>
+      </ScaffoldContainer>
+      <ScaffoldContainer className="flex flex-col gap-10" bottomPadding>
+        {!isPermissionsLoaded ? (
+          <GenericSkeletonLoader />
+        ) : !canReadAuthSettings ? (
+          <NoPermission isFullPage resourceText="access your project's authentication settings" />
+        ) : (
+          <>
+            <AdvancedAuthSettingsForm />
```

Comment:
same here - nit: existing problem on prod, the callout shows up prematurely similar to [here](https://github.com/supabase/supabase/pull/33335/files#r1942372443), but more obvious now since no scrolling required (occurs mainly on fresh tab session)

![image](https://github.com/user-attachments/assets/c740aa2b-1801-46b2-b4da-f194ac349174)